applications/crossbar/src/modules_v2/cb_phone_numbers_v2.erl

```diff
@@ -441,6 +442,61 @@ view_account_phone_numbers(Context) ->
     cb_context:set_resp_data(Context1, NewRespData).
 
 %% @private
+-spec maybe_add_port_request_numbers(cb_context:context()) -> kz_json:object().
+maybe_add_port_request_numbers(Context) ->
+    AccountId = cb_context:account_id(Context),
+    FilterState = kz_json:get_value(<<"filter_state">>
+                                   ,cb_context:query_string(Context)),
+    case check_filter_state(kz_util:to_binary(FilterState))
```

Comment:
I don't believe this will do. Any field of the response can be used as a filter. Not just state.
Maybe this could go into the function getting the view?